src/config/config.cc

```diff
@@ -659,6 +659,12 @@ Status Config::parseConfigFromPair(const std::pair<std::string, std::string> &in
     tokens[input.second] = input.first.substr(ns_str_size);
   }
   auto iter = fields_.find(field_key);
+
+  // Check if a configuration key is a valid key
```

Comment:
Can you explain why does it works here?

We write configs to `fields_`.
https://github.com/apache/incubator-kvrocks/blob/46e7eaf428c3e4935cc73f70be3f8c9066d5cb3c/src/config/config.cc#L226-L230

Reply:
Actually I wonder why not check `iter == fields_.end()` directly : )

Reply:
I was wondering what the properties of "fields" was. This was a very hacky fix, so I will dig deeper to see if I can make the code more "understandable" and less "magic" lol.